index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="./Components/Pages/Styles/main.css">
+  <title>Game Ranking: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./Components/Pages/Assets/icon-game.gif" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./Components/Pages/About/Component/about.html">About</a>
```

Comment:
- [ ] On the mobile version, the mobile menu, when opened, overlaps with the other elements on the page, making the text unreadable, please make sure to address this issue.

![](https://i.imgur.com/WI4U51x.png)

Reply:
Addressed 🙌